README.md

```diff
@@ -4,3 +4,9 @@
 
 1. Run `npm run lint` to check for lint errors
 2. Run `npm run lint-fix` to auto fix lint errors
+
+## How to run pa11y-ci checks
+
+1. Ensure any new views/pages have been added to .pallyci.json file.
```

Comment:
If you want to avoid having to manually add these a sitemap may also be used (would need to be added first though).

It might be something to create a new story around to enhance/further automate the accessibility testing https://www.npmjs.com/package/express-sitemap

Reply:
Going to add a new ticket for this during refinement